.travis.yml

```diff
@@ -1,18 +1,22 @@
-sudo: required
 language: java
 jdk:
-- oraclejdk8
+  - oraclejdk8
+
```

Comment:
Indentation can be overridden when I was experimented with coverity

Reply:
Not sure I'm following, this comment